src/markdown-engine.ts

```diff
@@ -305,10 +305,17 @@ export class MarkdownEngine {
     // protocal whitelist
     const protocolsWhiteList = this.config.protocolsWhiteList
       .split(",")
-      .map((x) => x.trim()) || ["http", "https", "atom", "file"];
+      .map((x) => x.trim()) || [
+      "http://",
+      "https://",
+      "atom://",
+      "file://",
+      "mailto:",
+      "tel:",
+    ];
```

Comment:
How about `defaultMarkdownEngineConfig.protocolsWhiteList` here? This will make it easier to update protocol list in future.